nodes/config/ui_page.html

```diff
@@ -59,15 +59,22 @@
                 typeField: '#node-input-layoutType'
             })
 
-            // backwards compatibility
-            if (this.visible === undefined || this.visible === 'true') {
+            // backwards compatibility - because NR can't do boolean values on dropdowns
+            if (this.visible === undefined || this.visible === true || this.visible === 'true') {
                 this.visible = true
                 $('#node-config-input-visible').val('true')
+            } else if (this.visible === false || this.visible === 'false') {
+                this.visible = false
+                $('#node-config-input-visible').val('false')
             }
```

Comment:
```suggestion
            if (this.visible === false || this.visible === 'false') {
                this.visible = false
                $('#node-config-input-visible').val('false')
            } else {
                this.visible = true
                $('#node-config-input-visible').val('true')
            }
```

while I realise by normal means `.visible` will be `undefined`, `"false"` or `false`, corruption and LLMs can (and do) put random guff like `null` or `"True"` as the value.

If we re-arrange the logic to have a "fallback `} else { ... }` then it will handle this case and others (and is somewhat more readable)